src/openrct2/ride/RideData.h

```diff
@@ -497,6 +498,8 @@ constexpr std::string_view MUSIC_OBJECT_SUMMER = "rct2.music.summer";
 constexpr std::string_view MUSIC_OBJECT_TECHNO = "rct2.music.techno";
 constexpr std::string_view MUSIC_OBJECT_WATER = "rct2.music.water";
 constexpr std::string_view MUSIC_OBJECT_WILD_WEST = "rct2.music.wildwest";
+constexpr std::string_view MUSIC_OBJECT_ACID = "openrct2.music.acid";
+constexpr std::string_view MUSIC_OBJECT_MECHANICAL = "rct2.music.mechanical";
```

Comment:
I saw something in another PR regarding waiting to add new music until some kind of fallback was established. Has that been established?

Reply:
I don't think it has, and personally, thinking about it. I am not sure if it's really necessary to have fallbacks. People can clear out the entire list of music if they wanted to. These rides are going to have to be cheated in at this point in time anyway. I don't think it's too terrible to have them be empty if the music object isn't available.

Though perhaps similarly to the scenery groups, the ride object can select a music object when it uses it as default? Except you can still uncheck it again if you wanted to.